css-logical-1/Overview.bs

```diff
@@ -204,11 +204,11 @@ Logical Values for the 'caption-side' Property</h3>
 
   <pre class="propdef partial">
     Name: caption-side
-    New values: inline-start | inline-end
+    New values: inline-start | inline-end | block-start | block-end
     Computed value: specified keyword
   </pre>
 
-  These two values are added only for implementations that support
+  These two inline values are added only for implementations that support
```

Comment:
The two inline values...

Reply:
It refers to `inline-start` and `inline-end`.

Reply:
Very confusing to start with "these" if you are only referring to some of them, something like this is clearer:

```suggestion
  The ''caption-side/inline-start'' and ''caption-side/inline-end'' values
  are added only for implementations that support
```

Reply:
Much better, thank you.